builder/frameworks/_embed_files.py

```diff
@@ -110,14 +110,14 @@ def transform_to_asm(target, source, env):
                 " ".join(
                     [
                         "riscv32-esp-elf-objcopy"
-                        if mcu in ("esp32c3", "esp32c6")
+                        if mcu in ("esp32c3", "esp32c5", "esp32c6")
```

Comment:
this (and the following) repeated lists of mcu feels like it could be refactored into each board json, or at least centralized into a function that said whether the mcu is riscv32 or not.